installer/host/qt_host_installer/io.h

```diff
@@ -13,7 +13,7 @@ namespace io
     bool writeImage(QString devicePath, QString deviceImage, QObject* caller=NULL);
     bool unmount(QString devicePath, bool isDisk = false);
     void updateKernelTable();
-    bool mount(QString devicePath, QString mountDir);
```

Comment:
but it's the same interface for linux, I migrated io_linux to follow this interface. would you rather have two distinct implementations?